lib/tasks/jenkins.rake

```diff
@@ -3,7 +3,7 @@ begin
 
   namespace :jenkins do
     task :unit => ['jenkins:setup:minitest', 'rake:test:units', 'rake:test:lib', 'rake:test:functionals']
-    task :integration => ["jenkins:setup:minitest", 'rake:test:integration']
+    task :integration => ['webpack:compile', 'jenkins:setup:minitest', 'rake:test:integration']
```

Comment:
good point, I was running it manually locally, moving this into the integration test helper.